app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, old_file, new_file = command.split(" ")
+    if old_file == new_file:
+        return
+    new_dir = os.path.dirname(new_file)
+    if new_dir != "":
```

Comment:
In Python, you can use a simple condition to check if a string is not empty.
```suggestion
    if new_dir:
```